plugins/labhub.py

```diff
@@ -136,9 +136,18 @@ def create_issut_cmd(self, msg, match):
         repo_name = match.group(1)
         iss_title = match.group(2)
         iss_description = match.group(3) if match.group(3) is not None else ''
+
+        try:  # This is gitter backend specific
+            link = '\nOpened via [gitter](https://gitter.im/{}?at={})'.format(
```

Comment:
Add username as well, of the user that created the issue, so he remains in the loop